modules/virt-configuring-rate-limiters.adoc

```diff
@@ -0,0 +1,18 @@
+
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-vm-control-plane-tuning.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-rate-limiters_{context}"]
+= Configuring rate limiters
+
+To compensate for large-scale burst rates, scale the `QPS`(Queries per Second) and `burst` `RateLimits` to process a higher rate of client requests or API calls concurrently for each component.
```

Comment:
Suggest changing `burst` `RateLimits` to "burst rate limits". There is no need to mark these terms in backticks in a generic context.

Reply:
What do you think about "...scale `QPS (Queries per Second) and `burst` values to process..."?